src/amo/reducers/viewContext.js

```diff
@@ -0,0 +1,32 @@
+/* @flow */
+import {
+  ADDON_TYPE_EXTENSION,
+  ADDON_TYPE_THEME,
+  VIEW_CONTEXT_EXPLORE,
+  VIEW_CONTEXT_HOME,
+  SET_VIEW_CONTEXT,
+} from 'core/constants';
+
+
+export type ViewContextType = {|
+  context: ADDON_TYPE_EXTENSION | ADDON_TYPE_THEME | VIEW_CONTEXT_EXPLORE |
```

Comment:
Ah, thanks for the info. Strangely even if I changed them to strings it still didn't work?

That syntax is obnoxious 😢